dags/brz_kr_etf_daily.py

```diff
@@ -0,0 +1,175 @@
+import json
+import xml.etree.ElementTree as ET
+from datetime import datetime, timedelta
+
+import requests
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.dummy import DummyOperator
+from airflow.operators.python import BranchPythonOperator, PythonOperator
+from airflow.providers.amazon.aws.operators.s3 import S3CreateObjectOperator
+from airflow.utils.trigger_rule import TriggerRule
+
+
+def _is_holiday_today(year, month, today):
+    url = (
+        "http://apis.data.go.kr/B090041/openapi/service/SpcdeInfoService/getRestDeInfo"
+    )
+    params = {
+        "solYear": year,
+        "solMonth": month,
+        "serviceKey": Variable.get("data_kr_service_key"),
+    }
+
+    response = requests.get(url, params=params)
+    root = ET.fromstring(response.text)
+    # 모든 <locdate> 태그의 값 추출
+    locdates = [item.text for item in root.findall(".//locdate")]
+
+    if int(month) == 12:
+        last_weekday = datetime(year, 12, 31)
+
+        while last_weekday.weekday() > 4:
+            last_weekday -= datetime.timedelta(days=1)
+
+        last_weekday_nodash = last_weekday.strftime("%Y%m%d")
+        locdates.append(last_weekday_nodash)
+
+    # today와 일치하는 값이 있는지 확인
+    if today in locdates:
+        return True
+    else:
+        return False
+
+
+def choose_api_task(ds_nodash):
+    date = datetime.strptime(ds_nodash, "%Y%m%d")
+    year = date.year
+    month = date.strftime("%m")
```

Comment:
P3; 오잉? 이렇게도 됩니다. year, month, day, hour, minute, second, microsecond 다 있어요~
```suggestion
    month = date.month
```

Reply:
아 이게 month를 두글자로 뽑으려고 저렇게 된건데 코드로 보니까 이상하네요!
date.month로 먼저 뽑아 놓고 요청 인자로 넣을때 두글자 스트링으로 변경하는 것으로 수정해보겠습니다.
(아래 리뷰도 참고해서 date 통으로 넘긴 후 함수 안에서 처리하겠습니다)

Reply:
아 맞다 ㅠㅠ 그렇군요. 그럼 지금 형태로도 괜찮습니다!